multiqc/modules/samtools/stats.py

```diff
@@ -128,7 +128,7 @@ def parse_samtools_stats(self):
             plot_html = plots.beeswarm.plot(self.samtools_stats, keys,
                                             {'id': 'samtools-stats-dp'})
             self.sections.append({
-                'name': 'Samtools Stats',
+                'name': 'Alignment metrics from samtools stats',
```

Comment:
Maybe also the other section in this module as `Percent Mapped` instead of `Alignment`?